.travis.yml

```diff
@@ -13,9 +13,6 @@ cache:
 # See: https://github.com/mozilla/addons-frontend/issues/3034
 install: yarn install --pure-lockfile
 
-matrix:
-  fast_finish: true
```

Comment:
When I looked at the travis manual, the setting was only shown for `matrix` so I removed it to make sure subsequent builds would pass. I now see that it works for `jobs` too, so we can likely add it back..